packages/node_modules/@ciscospark/plugin-avatar/src/avatar-url-batcher.js

```diff
@@ -53,6 +53,9 @@ const AvatarUrlBatcher = Batcher.extend({
   },
 
   handleItemSuccess(item) {
+    const maxAge = item.response.cacheControl && item.response.cacheControl.indexOf(`max-age=`) !== -1 && Number(item.response.cacheControl.split(`max-age=`)[1]) || undefined;
+    item = {uuid: item.uuid, size: item.size, url: item.response.url, maxAge};
+
     return this.getDeferredForResponse(item)
```

Comment:
I seem to recall the item itself has a cache property; I'm not sure the cache header is the appropriate place to pull the cache data from (since this is a bulk request, it actually seems to me that it's a bug that the response has a cache header at all).